components/library/libgsf/Makefile

```diff
@@ -8,7 +8,6 @@
 # source.  A copy of the CDDL is also available via the Internet at
 # http://www.illumos.org/license/CDDL.
 #
-
```

Comment:
Avoid removing this line

Reply:
Does it trigger something in the build system?

Reply:
No, just standard header format from https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/templates/configure.mk